package.json

```
@@ -4,7 +4,7 @@
   "description": "Self-hosted audiobook and podcast server",
   "main": "index.js",
   "scripts": {
-    "dev": "node index.js",
+    "dev": "nodemon index.js",
```

```suggestion
    "dev": "nodemon --watch server index.js",
```
Only need to watch files inside /server since /client is watched by Nuxt.